server/connection/socket.js

```diff
@@ -0,0 +1,93 @@
+const Websocket = require('ws');
+const {
+  createChat,
+  getReciveChatRoomInfo,
+  plusSellerUnreadCount,
+  plusBuyerUnreadCount,
+  updateLastChat,
+} = require('../data/chat');
+
+const rooms = new Map();
+
+class Socket {
+  constructor(server) {
+    this.wss = new Websocket.Server({
+      server,
+      verifyClient: async (info, cb) => {
+        const parsedUrl = info.req.url.split('/');
+        const [productId, buyerId, sellerId] = [parsedUrl[2], parsedUrl[3], parsedUrl[4]];
+        const roomId = parseInt(String(productId) + String(buyerId));
+        // info.req.user = sellerId;
+        info.req.user = Math.random() * 100;
+        info.req.room = roomId;
+        const roomInfo = await getReciveChatRoomInfo(roomId);
+        if (roomInfo.buyer == buyerId || roomInfo.seller == sellerId) {
```

Comment:
=== 이 아니라 == 을 사용하는 이유가 궁금합니다. 의도적으로 사용하신 건가요?
```suggestion
        if (roomInfo.buyer === buyerId || roomInfo.seller === sellerId) {
```

Reply:
테스트용으로 느슨하게 돌리려다보니 
확인을 못했네요 수정하도록 하겠습니다!ㅠㅠ